app/generator/content/schedule/part3/get-table.js

```diff
@@ -10,27 +12,29 @@ const getTable = (schedule) => {
     fillColor: '#d9d9d9',
```

Comment:
Can you add missing test checking the fill value of the table

Reply:
Sample of generated schedule was presented to the team on 12/06/2023, it was agreed that there should be no filled value, so, it has been removed, hence no test is required.